src/screens/send/components/fee-advanced/fee-advanced.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { View, TextInput } from 'react-native';
+import { Text } from '../../../../library';
+import stylesProvider from './styles';
+import { withTheme, IThemeProps } from '../../../../core/theme/with-theme';
+import { translate } from '../../../../core/i18n';
+import { FeeTotal } from '../fee-total/fee-total';
+import BigNumber from 'bignumber.js';
+import { Blockchain } from '../../../../core/blockchain/types';
+import { getBlockchain } from '../../../../core/blockchain/blockchain-factory';
+
+export interface IExternalProps {
+    gasPrice: BigNumber;
+    gasLimit: BigNumber;
+    blockchain: Blockchain;
+    onInputFees: (gasPrice: BigNumber, gasLimit: BigNumber) => any;
+}
+interface IState {
+    inputGasPrice: string;
+    inputGasLimit: string;
+}
+export class FeeAvancedComponent extends React.Component<
+    IExternalProps & IThemeProps<ReturnType<typeof stylesProvider>>,
+    IState
+> {
+    constructor(props: IExternalProps & IThemeProps<ReturnType<typeof stylesProvider>>) {
+        super(props);
+        const blockchainInstance = getBlockchain(this.props.blockchain);
+
+        this.state = {
+            inputGasPrice: blockchainInstance.account
```

Comment:
since `props.gasPrice` is a `BigNumber` you don't need to do `new BigNumber(Number(props.gasPrice))`